### DISCUSSION TITLE
I propose to make it so that Windows Defender can be disabled and not completely eliminated

### DISCUSSION THREAD

**Discussion by RobyReal12** [ionuttbara/windows-defender-remover#161] [Ideas]
Also, I hope you can implement the function to restore the choice to disable Windows Defender allowing it to be reactivated again, just as you presented the project on the github home page of your work.

**Comment by BeegDeek** [comment]
Why would you want Defender? Because Microsoft made it? Trust me Defender is garbage. I've been using Windows since 1995. All it does is constantly hog CPU cycles. Its like any other crap software program. If it doesn't do the job you uninstall it forever. When you go to any antivirus rating website it's not even listed
